divinediscipline/send-it
# #161913794  implement LFA UI feedback

**What does this PR do?**
implement LFA feedback

**Description of task to be completed**
- centralize dashboard cards
- move new order button to top
- add profile image avatar
- add footer to profile page and reduce footer size
- add dropdown on link to user's profile and log out

**How should this be manually tested**
navigagte to the application URL and view the front end pages to see the changes

**Any background context you want to provide?**
None

What are the relevant pivotal tracker stories?
 #161913794
